test/JDBC/input/BABEL_OBJECT_NAME-vu-prepare.mix

```diff
@@ -0,0 +1,74 @@
+-- To test table, trigger, procedure, function
+-- tsql
+CREATE TABLE babel_object_name_t1 (a int);
+GO
+
+CREATE TRIGGER babel_object_name_trg ON babel_object_name_t1 AFTER INSERT AS SELECT 1;
```

Comment:
we should create object names with lowercase, uppercase and mixed case

Reply:
addressed